packages/dashmate/configs/migrations.js

```diff
@@ -579,16 +579,17 @@ module.exports = {
     Object.entries(configFile.configs)
       .forEach(([, config]) => {
         // Update ports
-        config.platform.dashmate.helper.api.port = systemConfigs.base.platform
-          .dashmate.helper.api.port;
+        config.dashmate.helper.api.port = systemConfigs.base.dashmate.helper.api.port;
 
-        if (config.group === 'local') {
-          config.set('platform.drive.tenderdash.pprof.port', systemConfigs.base.platform
-            .drive.tenderdash.pprof + (i * 100));
-          i++;
-        } else {
-          config.platform.drive.tenderdash.pprof = systemConfigs.base.platform
-            .drive.tenderdash.pprof;
+        if (config.platform) {
```

Comment:
We don't need to check for Platform because now it exists for all configs. Now we use `platform.enabled` to determine do we need to start Platform or not